tensorflow/compiler/xla/service/gpu/cudnn_batchnorm_rewriter.cc

```diff
@@ -78,13 +91,28 @@ Status Visitor::HandleBatchNormInference(HloInstruction* batch_norm) {
 
   std::vector<HloInstruction*> operands(batch_norm->operands().begin(),
                                         batch_norm->operands().end());
+
+  if (IsF32BatchNormWithFP16Inputs(batch_norm)) {
```

Comment:
Maybe just remember this in a local variable?

Reply:
I actually had it as a local variable in the earlier PR but changed it based on your comment :). 
Do u want something like this? (See line 99 onwards in the modified version) https://github.com/tensorflow/tensorflow/pull/32665/commits/5e8de43c588cee0ac510e35cfb62e91c734cca61#diff-f57e826cf2d3f0a9699cb0451fa78738

Reply:
Yes, but we need a more targeted name than `is_convert_inserted` since the fundamental property is that this is an fp16 batchnorm, not that a convert was inserted.

Reply:
I see...I read your earlier comment incorrectly. Sorry abt that.